### PR TITLE
Generate CLI docs.

### PR DESCRIPTION
**Pull request by wlynch**
#### Summary

Generate CLI docs for all cobra commands.

#### Release Note

NONE

#### Documentation

📖 📖 📖

### CODE REVIEW

**Comment by imjasonh** [comment]
From experience, you may want a CI check that fails if these docs haven't been updated.

`ko` has this: https://github.com/ko-build/ko/blob/main/hack/presubmit.sh#L29-L32

File: main.go [hunk 1]
```diff
@@ -35,8 +33,6 @@ func main() {
 	}
 
 	rootCmd := root.New(cfg)
-	rootCmd.AddCommand(version.New(cfg))
```
Comment (eddiezane) [inline main.go:38]: Any reason for this other than to keep main clean?

**Review by eddiezane — APPROVED** [review]
+1 to presubmit

File: main.go (hunk 1 above)
Comment (wlynch) [reply main.go:38]: Yeah - the doc generator needs the subcommands need to be added to pick them up. This moves the AddCommand's to the rootCmd creation to guarantee both main's have the same command tree.

**Review by eddiezane — APPROVED** [review]

**Pull request merged**